64.md

```diff
@@ -0,0 +1,65 @@
+NIP-64
+======
+
+Inbox model
+-----------
+
+`draft` `optional`
+
+Suppose **Sarah** wants to subscribe to the notes **Walter** writes.
+
+She must let Walter know she is following him. The first step is to find Walter's _inbox list_:
+
+```jsonc
+{
+  "kind": 10064,
```

Comment:
Inboxes and outboxes are both just relays in reality. Put another way, an outbox is a relay with a "write" marker, and an inbox is a relay with a "read" marker. So this nip can reuse kind 10002 events from NIP-65.

Reply:
I think we gain nothing by conflating two different kinds of relays. Relays specified on kind 10002 are already there and already being used. Relays that implement this NIP would require a little bit of special behavior, so it's better to signal them properly.

Also I think it's very important that users announce that they support this NIP before others start engaging with them through it, having a special kind published serves that purpose.

Reply:
> it's very important that users announce that they support this NIP

After #991 gets merged, you can pick a new relay usage flag to mean "nip64 relay" instead of introducing `kind:10064`.

Reply:
We could use a different tag in kind 10002 if people prefer that, but I honestly thinks it's always better to use a different event kind when the use case and implementation contexts are different. It's not a super strong argument, more like a rule-of-thumb I came up with, same as I said in https://github.com/nostr-protocol/nips/pull/686#discussion_r1536542654